webview-ui/src/components/chat/TaskHeader.tsx

```diff
@@ -351,7 +351,7 @@ const TaskActions = ({ item }: { item: HistoryItem | undefined }) => (
 		<Button variant="ghost" size="sm" onClick={() => vscode.postMessage({ type: "exportCurrentTask" })}>
 			<span className="codicon codicon-cloud-download" />
 		</Button>
-		{item?.size && (
+		{!!item?.size && item.size > 0 && (
```

Comment:
I noticed this errant "0" in the bottom right of my task header:
![Uploading Screenshot 2025-02-25 113946.png…]()